public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -273,6 +279,14 @@ export function AnomalyResults(props: AnomalyResultsProps) {
     }
   };
 
+  const getInitProgressMessage = () => {
+    return detector && isDetectorInitializing && detector.initProgress
+      ? `The detector needs to capture approximately
+                  ${detector.initProgress.neededShingles} data points for initializing. If your data stream is continuous, this process will take around
+                  ${detector.initProgress.estimatedMinutesLeft} minutes; if not, it may take even longer. `
```

Comment:
Just confirm, is the detector state based on how many data points in the tree? I remember our testing detector on NAB data will become running quickly before capturing 128 data points as cold start will build RCF model first. I'm asking this to make sure the detector state, neededShingles and estimatedMinutesLeft can match.

Reply:
> is the detector state based on how many data points in the tree

yes

The number would be the total data points in the tree before cold start, and needed shingles after cold start.  In your NAB example, we should only see this message briefly and the progress bar should disappear as initialization is finished.